packages/patternfly-4/react-charts/src/components/ChartBullet/ChartBullet.tsx

```diff
@@ -720,12 +715,16 @@ export const ChartBullet: React.FunctionComponent<ChartBulletProps> = ({
     // Adjust for padding
     if (legendPosition === ChartLegendPosition.bottom) {
       dy = horizontal
-        ? defaultPadding.top * .5 + (defaultPadding.bottom * .5 - (defaultPadding.bottom)) - 25
-        : title ? -defaultPadding.bottom + 60 : -defaultPadding.bottom;
+        ? defaultPadding.top * 0.5 + (defaultPadding.bottom * 0.5 - defaultPadding.bottom) - 25
+        : title
+        ? -defaultPadding.bottom + 60
+        : -defaultPadding.bottom;
     } else if (legendPosition === ChartLegendPosition.bottomLeft) {
       dy = horizontal
-        ? defaultPadding.top * .5 + (defaultPadding.bottom * .5 - (defaultPadding.bottom)) - 25
-        : title ? -defaultPadding.bottom + 60 : -defaultPadding.bottom;
+        ? defaultPadding.top * 0.5 + (defaultPadding.bottom * 0.5 - defaultPadding.bottom) - 25
+        : title
+        ? -defaultPadding.bottom + 60
+        : -defaultPadding.bottom;
```

Comment:
This indentation looks a little off?

Reply:
It looks like Prettier will indent a nested ternary if it is nested within the true-branch (`foo ? bar ? 1 : 2 : 3`) but not if it is trailing / nested in the false-branch (`foo ? 1 : bar ? 2 : 3`). This is a bit weird but seems reasonable to me... to be honest though, maybe we should avoid nested ternary expressions unless we use parentheses to make them more clear. Maybe that would help here.

Reply:
Looks like this is a subject of debate in Prettier: https://github.com/prettier/prettier/issues/5814

Reply:
Looks like this is a subject of debate in Prettier: prettier/prettier#5814

Reply:
Huh. Adding parens around the second expression there, Prettier removes them. Maybe that's a case where `// prettier-ignore` would be useful if we need an exception to make that code extra clear.

Reply:
I didn't realize `// prettier-ignore` was available. That could be quite useful for those couple of weird issues

Reply:
Agreed. You can even configure a `.prettierignore` file to ignore entire files/directories if necessary